pkg/controller/apischeme/apischeme_controller.go

```diff
@@ -303,6 +298,11 @@ func SetAPISchemeStatus(crObject *cloudingressv1alpha1.APIScheme, reason, messag
 		message,
 		utils.UpdateConditionNever)
 	crObject.Status.State = ctype
+
+	err := r.client.Status().Update(context.TODO(), crObject)
+	if err != nil {
+		log.Error(err, "Error updating cr status")
+	}
```

Comment:
I get that you made this logically the same as it was before, which is cool, but perhaps add a TODO to bubble this error up to the caller? Especially in the "Success" branch (L242), I would think we would want to react differently if we couldn't update the status.

Reply:
Possibly, but I wanted to preserve existing behavior. Will add a TODO.